### ISSUE TITLE
git status doesn't refresh in new project `git init`ed under vsCode

### ISSUE THREAD

**Issue by estaub** [microsoft/vscode#102234]
- VSCode Version: 1.47.0
- OS Version: MacOs 10.15.5

Steps to Reproduce:

Not sure.  This may clarify what happened in https://github.com/microsoft/vscode/issues/90670 .  I'm just reporting to help out.  I'm new to vsCode.

I created a new directory, opened vsCode on it, and did a `git init` in the terminal.  I created a bunch of files, and eventually created a .gitignore.  When I finally got to wanting to check in, I saw that the Source Control pane "Changes" list was listing all of the stuff in .gitignore with a "U" (untracked).  On the other hand, the "Explorer" panel correctly grayed the ignored directories and files.

When I attempted to do an "Add" of everything from vsCode, git (correctly) errored as in #909670, complaining that I was trying to add ignored files.  

I then did a `git add .` and `git commit -m "initial"` from the terminal, which worked fine.  However, vsCode's view of all the file statuses remained unchanged.

I closed and reopened the workspace, and everything was then fine, and updates correctly.

Does this issue occur when all extensions are disabled?: Sorry, don't know.

**Comment by nepetadosmil** [comment]
Could it possibly be related to #101507?

**Comment by estaub** [comment]
@nepetadosmil Seems very unlikely, but not impossible.

**Comment by joaomoreno** [comment]
So after committing everything works as expected?

**Comment by estaub** [comment]
@joaomoreno Yes, all is fine (AFAIK) after committing.

**Comment by estaub** [comment]
> So it seems that git simply didn't pick up the changes fast enough.  

No, this was at least 15 minutes, probably much longer, as I tried to figure out what (as a vsCode newbie) I had misconfigured or otherwise done wrong. There's probably some other condition required for it to occur.  

(No need to reopen - just giving info.)
